File: back-end/skeleton/nodejs-api-serverless/src/entities/Role.ts
import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
import BaseEntity from './Base';

@Entity('Role')
export default class Role extends BaseEntity {
    @PrimaryGeneratedColumn({ name: 'RoleId' })
    roleId: number;

    @Column({ name: 'Name' })
    name: string;

    @Column({ name: 'Description' })
    description?: string;
}

File: back-end/skeleton/dotnet/src/DotnetSkeleton.API/DotnetSkeleton.Core.Domain/Entities/MySQLEntities/Role.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using DotnetSkeleton.SharedKernel.Utils.Models.Entities;

namespace DotnetSkeleton.Core.Domain.Entities.MySQLEntities
{
    /// <summary>
    /// Role entity
    /// </summary>
    [Table(nameof(Role))]
    public class Role : BaseEntity
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int RoleId { get; set; }
        public required string Name { get; set; }
        public string? Description { get; set; }
    }
}